### ISSUE TITLE
db-migrate-pg dependency should be specified clearly

### ISSUE THREAD

**Issue by gokul-hashlearn** [db-migrate/english-docs#16]
Installation instructions should specify that 'db-migrate-pg' module needs to be installed separately, if postgres is the target database.
##

**Comment by wzrdtales** [comment]
Yes you're right, if you want PRs are appreciated, otherwise I placed this already onto the todo pipeline (zenhub).

Thank you for reporting!
